[ZT] Access policy builder supports SCIM groups (#18767)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/policies/access/index.mdx

| Any Access Service Token | The request will need to present the headers for any [service token](/cloudflare-one/identity/service-tokens/) created for this account.                                      | ✅               | ✅                               |
| Login Methods            | Checks the identity provider used at the time of login.                                                                                                                       | ✅               | ❌                               |
| Authentication Method    | Checks the [multifactor authentication](/cloudflare-one/policies/access/mfa-requirements/) method used by the user, if supported by the identity provider.                    | ✅               | ❌                               |

> | Identity provider group  | Checks the user groups configured with your identity provider (IdP). This selector only displays if you use Microsoft Entra ID, GitHub, Google, or Okta as your IdP, or if you provision the IdP groups using [SCIM](/cloudflare-one/identity/users/scim/).      | ✅               | ❌                               |

Author: ```suggestion
| Identity provider group  | Checks the user groups configured with your identity provider (IdP). This selector only displays if you use Microsoft Entra ID, GitHub, Google, Okta, or an IdP that provisions groups with [SCIM](/cloudflare-one/identity/users/scim/).      | ✅               | ❌                               |
```
